modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -274,7 +327,7 @@ def data_attributes
       form_data&.dig('data', 'attributes')
     end
 
-    def veteran
+    def veteran_data
```

Comment:
Renamed so not to conflict with the new veteran method that creates an Appellant